<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QGroupBox" name="groupBoxSort">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>100</height>
             </size>
            </property>
            <property name="title">
             <string>排序</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>排序字段</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1" colspan="2">
              <widget class="QComboBox" name="comboFields">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QRadioButton" name="rbAsc">
               <property name="text">
                <string>升序</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QRadioButton" name="rbDesc">
               <property name="text">
                <string>降序</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBoxFilter">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>100</height>
             </size>
            </property>
            <property name="title">
             <string>数据过滤</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QRadioButton" name="rbMan">
               <property name="text">
                <string>男</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbWoman">
               <property name="text">
                <string>女</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbBoth">
               <property name="text">
                <string>全部</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>表</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QTableView" name="tableView"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="baseSize">
        <size>
         <width>800</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>详细资料</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>工号</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="dbSpinEmpNo">
            <property name="minimum">
             <number>2000</number>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2" rowspan="6">
           <widget class="QGroupBox" name="groupBox_5">
            <property name="title">
             <string>照片</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="dbLabPhoto">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>姓名</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="dbEditName"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>性别</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="dbComboSex">
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>身高</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="dbSpinHeight">
            <property name="maximum">
             <double>3.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>出生日期</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QDateEdit" name="dbEditBirth"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>手机号</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="dbEditMobile"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>出生省份</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QComboBox" name="dbComboProvince">
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="2" rowspan="5">
           <widget class="QGroupBox" name="groupBox_6">
            <property name="title">
             <string>备注</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QPlainTextEdit" name="dbEditMemo"/>
             </item>
            </layout>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>城市</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="dbEditCity"/>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>部门</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QComboBox" name="dbComboDep">
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>学历</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QComboBox" name="dbComboEdu">
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>工资</string>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QSpinBox" name="dbSpinSalary">
            <property name="maximum">
             <number>20000</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>772</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>操作</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actAppend"/>
    <addaction name="actInsert"/>
    <addaction name="actDelete"/>
    <addaction name="separator"/>
    <addaction name="actSetPhoto"/>
    <addaction name="actClearPhoto"/>
    <addaction name="actSalary"/>
    <addaction name="separator"/>
    <addaction name="actSubmit"/>
    <addaction name="actRevert"/>
    <addaction name="separator"/>
    <addaction name="actQuit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>数据源</string>
    </property>
    <addaction name="actOpenDB"/>
    <addaction name="actOpenTable"/>
    <addaction name="actOpenQuery"/>
   </widget>
   <addaction name="menu_2"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actOpenDB"/>
   <addaction name="separator"/>
   <addaction name="actOpenTable"/>
   <addaction name="actOpenQuery"/>
   <addaction name="actRelation"/>
   <addaction name="separator"/>
   <addaction name="actAppend"/>
   <addaction name="actInsert"/>
   <addaction name="actDelete"/>
   <addaction name="separator"/>
   <addaction name="actSetPhoto"/>
   <addaction name="actClearPhoto"/>
   <addaction name="actSalary"/>
   <addaction name="separator"/>
   <addaction name="actSubmit"/>
   <addaction name="actRevert"/>
   <addaction name="separator"/>
   <addaction name="actQuit"/>
  </widget>
  <action name="actOpenDB">
   <property name="text">
    <string>打开数据库</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actAppend">
   <property name="text">
    <string>添加数据</string>
   </property>
  </action>
  <action name="actInsert">
   <property name="text">
    <string>插入数据</string>
   </property>
  </action>
  <action name="actSubmit">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actRevert">
   <property name="text">
    <string>取消</string>
   </property>
  </action>
  <action name="actDelete">
   <property name="text">
    <string>删除记录</string>
   </property>
  </action>
  <action name="actSetPhoto">
   <property name="text">
    <string>设置照片</string>
   </property>
  </action>
  <action name="actClearPhoto">
   <property name="text">
    <string>清除照片</string>
   </property>
  </action>
  <action name="actSalary">
   <property name="text">
    <string>调整工资</string>
   </property>
  </action>
  <action name="actOpenTable">
   <property name="text">
    <string>表</string>
   </property>
  </action>
  <action name="actOpenQuery">
   <property name="text">
    <string>查询</string>
   </property>
  </action>
  <action name="actRelation">
   <property name="text">
    <string>相关表</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
